README.md

```diff
@@ -17,15 +17,11 @@ version 3 first with one of many available converters._
 
 ## Why This?
 
-The Python clients generated by openapi-generator support Python 2 and therefore come with a lot of baggage. This tool
-aims to generate clients which:
+The Python clients generated by openapi-generator comes some downsides. This tool aims to improve on it by:
 
-1. Use all the latest and greatest Python features like type annotations and dataclasses
-1. Don't carry around a bunch of compatibility code for older version of Python (e.g. the `six` package)
-1. Have better documentation and more obvious usage instructions
-
-Additionally, because this generator is written in Python, it should be more accessible to contribution by the people
-using it (Python developers).
+1. Generating clients which use all the latest and greatest Python features like type annotations and dataclasses.
+2. Having better documentation and more obvious usage instructions.
+3. Being easier to improve and extend, especially for python developers. Since openapi-python-client is written in python and jinja2 templates, while openapi-generator is a larger Java project which uses more restrictive mustache templates.
```

Comment:
```suggestion
1. Being written in Python, making it easier to improve and extend for Python developers. It's also much easier to install and use if you already have Python.
```

Reply:
I'd prefer to leave in something about jinja vs mustache templating. My impression was that jinja was the most popular templating language for python, Mustache is fairly popular for javascript but much less so in the python world. Also jinja is more flexible/powerful since it doesn't try to be logicless

Reply:
I'd like to leave in something about jinja vs Mustache templating. Jinja seems to be the prefered python template framework, mustache seems to be relatively popular in javascript land but not in python land. Also jinja not trying to be logicless is more powerful/flexible then Mustache